master/buildbot/test/unit/test_process_build.py

```diff
@@ -385,11 +385,11 @@ def claim(owner, access):
 
         b.startBuild(FakeBuildStatus(), None, self.workerforbuilder)
 
-        self.assert_(('startStep', (self.workerforbuilder.worker.conn,), {})
-                     not in step.method_calls)
+        self.assertNotIn(('startStep', (self.workerforbuilder.worker.conn,), {}),
+                     step.method_calls)
         self.assertEquals(claimCount[0], 1)
-        self.assert_(b.currentStep is None)
-        self.assert_(b._acquiringLock is not None)
+        self.assertTrue(b.currentStep is None)
```

Comment:
buildbot master is now python2.7 only, so You have assertIsNone assertRaises, and all that fancy stuff.
Note that even when we were compatible 2.6, we added some of them via monkey patching (we might want to remove those btw)